R/interp.R

```diff
@@ -1,8 +1,10 @@
 #' Interpolate a formula
 #'
-#' Interpolation replaces sub-expressions of the form \code{UQ(x)} with
-#' the evaluated value of \code{x}, and inlines sub-expressions of
-#' the form \code{UQS(x)}.
+#' Interpolation replaces sub-expressions of the form \code{UQ(x)}
+#' with the evaluated value of \code{x}, and inlines sub-expressions
+#' of the form \code{UQS(x)}. Syntactic shortcuts are provided for
+#' unquoting and unquote-splicing by prefixing with \code{!!} and
+#' \code{!!!}.
```

Comment:
Ok, let's just go with it. I can't see an option that's better (unless we use `!!.` or `!!?`)